package.js

```diff
@@ -3,13 +3,20 @@ Package.describe({
   name: "practicalmeteor:mocha",
   summary: "Write package tests with mocha and run them in the browser or from the command line with spacejam.",
   git: "https://github.com/practicalmeteor/meteor-mocha.git",
-  version: '2.1.0_7'
+  version: '2.1.1-rc.1',
+  testOnly: true
+});
+
+
+Npm.depends({
```

Comment:
Although it doesn't look like from this PR that I actually ever switch out the use of the bundled file for an `Npm.depends()`? I'm not sure what's happened, this PR seems kind of garbled.